### ISSUE TITLE
Change breaks with the non-unicode InnoSetup 5.x version... it cant read the iss file.

### ISSUE THREAD

**Issue by fvarrui** [fvarrui/JavaPackager#297]
FYI: This change breaks with the non-unicode InnoSetup 5.x version... it cant read the iss file. 

This IS version is old, but this could be added to the documentation. (use 6.x or 5.x Unicode)

_Originally posted by @commi in https://github.com/fvarrui/JavaPackager/issues/271#issuecomment-1427887170_

**Comment by lhDream** [comment]
We can increase the configuration and set not to write BOM. 👀

**Comment by commi** [comment]
That would also work... but since the iss scripts are written in UTF-8, this can create garbled strings in IS 5. So maybe the BOM is a nice 'safeguard' against this:)
In any case you could add to https://github.com/fvarrui/JavaPackager/blob/master/docs/windows-tools-guide.md that IS 5 non-unicode is not supported, because there is just no way to support both.

**Comment by fvarrui** [comment]
`winConfig.iss.bom=true|false`?

**Comment by fvarrui** [comment]
I think a system that allows adding/deleting the BOM in the files generated from the Velocity templates would be more flexible, allowing you to select the template to which it's applied.

I mean:

```xml
<templates>
    <template>
        <name>iss.vtl</name>
        <bom>true|false</name>
        [here we can add new options for future features related to templates]
    </template>
</templates>
```

**Comment by fvarrui** [comment]
Work in progress in branch `issue-297`

**Comment by fvarrui** [comment]
Just published `1.7.3-20230701.111610-13` snapshot version.

How to disable UTF-8 with BOM for `iss` files:

```xml
<templates>
    <template>
        <name>windows/iss.vtl</name>
        <bom>false</bom>
    </template>
</templates>
```

> UTF-8 with BOM is enabled by default just as it was before.

**Comment by fvarrui** [comment]
Can we release this feature in 1.7.4?

**Comment by commi** [comment]
I see no reason why not:)

**Comment by fvarrui** [comment]
Branch issue-297 merged into devel, ready to be released in v1.7.4

**Comment by fvarrui** [comment]
1.7.4 released to Maven Central
